docs/QUICK_START.md

```diff
@@ -0,0 +1,193 @@
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
+
+Once the firmware is installed, you can confirm operation by observing the LED.
+- When either a receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+- When the Tx module and receiver are both powered on and bound, a LED (usually green) should flash more slowly at 1 Hz (once per second, on for 1/2 second then off for 1/2 second repeatedly) to indicate they are connected and exchanging radio messages.
+
+## 5. Install the mLRS LUA script and configure your mLRS system
+
+Follow the [instructions](LUA.md) and copy the correct mLRS LUA script to the SCRIPTS\TOOLS folder on your SD card and setup your RC radio external module.
+
+Start mLRS Configurator LUA script from the SYS->TOOLS menu and confirm you can talk to the Tx module.  You should see the TX module firmware version reported at the top of the screen.
+
+Use the mLRS LUA script to Confirm the receiver is connected and the firmware versions are recent enough to support the features you need.
+- If the radio link does not connect, follow the [Binding](BINDING.md) procedure.
+- It is HIGHLY recommended to use the same firmware version on both the receiver and Tx module. The latest released firmware is also recommended, especially for new installs.  If your Rx and Tx module firmware versions don't match or are very much out of date, return to the previous step and install the latest mLRS firmware release.
+
+Use the mLRS LUA script to configure your mLRS system
+- Leave your Rx powered on and connected while you complete the following steps.
+- Set the correct RF Band for your region (If using 900 MHz hardware).
+- You can also change the Bind Phrase if desired.
+- Set your desired Mode
+- If you didn't connect sbus or CRSF RC output from your receiver to your flight controller, enter the "Edit Rx" page and set Snd RcChannels to "rc override" (or "rc channels" for ArduPilot 4.6 with RC Channels support).  Return to the main page.  If your radio has a black and white screen, you will need to use the [CLI](CLI.md) for this.
+- Select "Save" and verify that your Rx reconnects.
+- You may wish to review and adjust the Rx and Tx power settings as well as other settings at this time, but the defaults should be OK to get started.
+
+## 6. Optionally, Install the Yaapu Telemetry script
+
+You may wish to install the [Yaapu Telemetry script](https://github.com/yaapu/FrskyTelemetryScript) which uses the CRSF telemetry messages which mLRS generates on your Tx module from the MAVLink data it receives when the Rx Serial Link Mode is set to "MAVLink" or "MAVLinkX".
+
+- If you won't always use a ground station to set MAVLink stream rates when it starts up, be sure to set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.
```

Comment:
This seems out of place for Yappu?  Probably better in the FC Section?

Reply:
The mLRS docs don't seem to acknowledge this, but the ground station will request what it wants to display and the rates it requests by default generally work fine and are also configurable in the ground station.  Thus, this step is really only required if you are ever going to use Yaapu without also using a ground station and I don't think some users will even use Yaapu and I don't want to waste their time.  That's why it's here.  Should I try to make this more clear here or do we want to continue to imply that everyone needs to set rates in ArduPilot and set the bit to ignore the ground station requests?

Reply:
IMO it is better for folks to set stream rates as it then becomes a known state.  A few folks have gotten stuck because their stream rates were set to 0 (no idea how, the defaults should be all 1) and then said the CRSF telemetry / Yappu wasn't working.  I think this is just another 'encourage' situation like above for upgrading to latest version but you're right this isn't strictly necessary in a getting started guide.

Reply:
I think you may have that wrong.
I remember noticing that my stream rates are usually set to 0 in ArduCopter which doesn't matter since I always use a ground station and that sets reasonable rates at startup.  I just checked by explicitly resetting a test board to default parameters and the default stream rates are all 0.

Reply:
Oh wow - I never would've known this, ArduPlane handles it differently: 

![image](https://github.com/user-attachments/assets/dcfdf5b1-cd44-440d-a8a5-68b67ddf3941)

Reply:
Wow.  That just stupid.  Why should it be different?

Reply:
No idea :)